lib/node_modules/@stdlib/string/base/stickycase/docs/types/test.ts

```diff
@@ -0,0 +1,50 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import stickycase from './index';
+
+// TESTS //
+
+// The function returns a string...
+{
+	stickycase( 'hello world' ); // $ExpectType any
+}
+
+// The compiler throws an error if the function is provided a value other than a string...
+{
+	// eslint-disable-next-line expect-type/expect
+	stickycase( true ); // $ExpectError
+	// eslint-disable-next-line expect-type/expect
+	stickycase( false ); // $ExpectError
+	// eslint-disable-next-line expect-type/expect
+	stickycase( undefined ); // $ExpectError
+	// eslint-disable-next-line expect-type/expect
+	stickycase( 5 ); // $ExpectError
+	// eslint-disable-next-line expect-type/expect
+	stickycase( [] ); // $ExpectError
+	// eslint-disable-next-line expect-type/expect
+	stickycase( {} ); // $ExpectError
+	// eslint-disable-next-line expect-type/expect
+	stickycase( ( x: number ): number => x ); // $ExpectError
```

Comment:
Disabling the lint rule defeats the purpose of the tests.

```suggestion
	stickycase( true ); // $ExpectError
	stickycase( false ); // $ExpectError
	stickycase( undefined ); // $ExpectError
	stickycase( 5 ); // $ExpectError
	stickycase( [] ); // $ExpectError
	stickycase( {} ); // $ExpectError
	stickycase( ( x: number ): number => x ); // $ExpectError
```